routes/notes.router.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+	createNoteForVideo,
+	getNoteFromDb,
+	updateNote,
+	deleteNote,
+} = require('../controllers/notes.controller');
+
+router.route('/').post(createNoteForVideo);
```

Comment:
You can just write router.post('/', createNoteForVideo); if theres just one route. :D